website_snippet_contact_form/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import http
+from openerp.http import request
+from openerp.addons.website_crm.controllers.main import contactus as Contactus
+from openerp.exceptions import ValidationError
+
+
+class snippetContactus(Contactus):
+
+    @http.route(['/crm/contactus'], type='http', auth="public", website=True)
```

Comment:
How about separating this from the core controller? Maybe into `/crm/contactus/snippet`...